src/sql/_testing.py

```diff
@@ -198,6 +198,8 @@ def database_ready(
             eng.close()
             print(f"{database} is initialized successfully")
             return True
+        except ModuleNotFoundError:
+            raise
```

Comment:
Do we want to perform any additional operations here?

Reply:
no, we just need to raise the original exception. this allows the tests to fail quickly if they have missing dependencies

Reply:
got it!